pkg/test/constants/constants.go

```diff
@@ -46,6 +50,13 @@ var TestNameRegexesToSkip = []string{
 	// Disable due to TF bug https://github.com/hashicorp/terraform-provider-google/issues/16255.
 	// We can't specify labels in the create operation, that causes AssertLabelsMatchAndHaveManagedLabel check to fail.
 	".*(regionalforwardingrulepsc).*",
+	// Tests added for sci-fi direct controller is temporarily disabled.
```

Comment:
Hm, I'm slightly concerned that we may lose actual good signal about the direct resources. Can we consider just disabling the export part of the tests for this if it's easy to do ?

Game to do that as a follow on and just ignore for now but would love to hear from Yuwen too.

Reply:
+1, thinking about the same!